MAUI/Popup/popup-positioning.md

```diff
@@ -690,4 +690,28 @@ Download the entire source code from GitHub [here](https://github.com/Syncfusion
   </ContentPage.Content>
 </ContentPage>
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+### Position the popup over the action bar
+
+The SfPopup can be positioned without considering the action bar by using the `SfPopup.IgnoreActionBar` property. When set to true, this allows the popup to be positioned over the action bar without any constraints.
+
+{% tabs %}
+{% highlight xaml hl_lines="2" %}
+<syncfusion:SfPopup x:Name="popup"
+                    IgnoreActionBar="True"> 
+</syncfusion:SfPopup>
+{% endhighlight %}
+{% highlight c# hl_lines="2" %}
+SfPopup popup = new SfPopup();
+popup.IgnoreActionBar = True;
+popup.Show(0,0);
+{% endhighlight %}
+{% endtabs %}
+
+![Positioning the popup without considering action bar](Images/popup-positioning/maui-popup-popsitioning-without-actionbar-consideration.png)
+
+If IgnoreActionBar is set to false, the popup positioning will take the action bar into account, ensuring it does not overlap.

```

Comment:
highlight the API,
Remove unwanted image

Reply:
Modified